.github/dependabot.yml

```
@@ -9,3 +9,27 @@ updates:
     labels:
       - area/dependencies
       - area/ci
+  - package-ecosystem: npm
+    directory: /themes/src/main/resources/theme/keycloak/common/resources
+    schedule:
+      interval: weekly
+    open-pull-requests-limit: 999
```

Let's make sure that automatic rebasing is disabled so that we don't spam the CI every time a PR is merged.
```suggestion
    open-pull-requests-limit: 999
    rebase-strategy: disabled
```